disco/api/client.py

```diff
@@ -229,6 +229,12 @@ def channels_messages_reactions_delete(self, channel, message, emoji, user=None)
 
         self.http(route, obj)
 
+    def channels_messages_reactions_delete_all(self, channel, message):
+        route = Routes.CHANNELS_MESSAGES_REACTIONS_DELETE_ALL
+        obj = dict(channel=channel, message=message)
+
+        self.http(route, obj)
```

Comment:
Please format this like the rest of the functions in this file:

```suggestion
        self.http(Routes.CHANNEL_MESSAGES_REACTIONS_DELETE_ALL, dict(channel=channel, message=message))
```